### PR TITLE
HTML and CSS code cleanup and tweaks; correct favicon added

### PR DESCRIPTION
**Pull request by wUFr**

### CODE REVIEW

File: stream.html [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-            <a href="geeksabroad.html"><img alt="Geeks abroad" class="b1 btn" src="images/abroad.png"/></a>
-            <a href="https://groups.google.com/forum/#!forum/squirrel-army" target="_new"><img alt="Squirrel army" class="b2 btn" src="images/squirrel.png"/></a>
-            <a href="https://github.com/OSAlt" target="_new"><img alt="OS.ALT" class="b3 btn" src="images/osalt.png"/></a>
-            <a href="gaming.html"><img alt="Gaming" class="b4 btn" src="images/gaming.png"/></a>
-            <img alt="Streaming" class="b5 btn sel" src="images/youtube.png"/>
-            <a href="survey.html"><img alt="Skills Survey" class="b6 btn" src="images/joinus.png"/></a>
-        </div>
-    </div>
-    <div>
-        <iframe src="https://www.youtube.com/embed/gU3cong_zI0" class="gaming" frameborder="0" allowfullscreen></iframe>
```
Comment (safaci2000) [inline stream.html]: This is really good actually.  Though the URL was changed in master, mind doing a quick rebase and I'll merge this in so you get the credit for it.

**Comment by wUFr** [comment]
ok, if i didn't miss something, should be fixed :)

**Comment by wUFr** [comment]
sec, will add one more hotfix :D

**Comment by wUFr** [comment]
done

**Pull request merged**